### ISSUE TITLE
1.18.2 issue - hour formatting is wrong

### ISSUE THREAD

**Issue by xFirefalconx** [EwyBoy/Barista#5]
![grafik](https://user-images.githubusercontent.com/106652912/229671361-f024f1c4-e6c9-4051-9094-5d836758e656.png)
As you can see, it counts minutes to over 60 so I believe this is an issue. 
I will also open another issue for my other problem.

**Comment by xFirefalconx** [comment]
It's been a while since I reported this issue, and I observed that on 1.18.2 it simply doesn't reduce the number of the minutes by 60 when increasing the hour by 1. So the easiest fix is to 
(pseudocode)
```
when (hour increases+1)
   {
       minutes-60;
   }
``` 
Or the sort I have no clue about the code right now. 
In regard of no update so far since April, I will contemplate to attempt to fork and fix this since it's MIT license but hopefully you come back to it _before I do_ and update the 1.18.2 if you happen to have done this for others already (yes/no?). 

**Please just let me know** what to expect so I can work/wait on the better solution for my modpack. It looks quite dorky to have a strange number that doesn't add up on the window bar.

**Comment by EwyBoy** [comment]
Hey, sorry! Been taking a break from modding due to personal reasons and been logged out of my github account for the past couple of months.
I will create a fix for this once I return after new year. Feel free to fork my projects any time.
Sorry for the trouble this has caused. It is a stupid oversight on my part and should be a simple fix.

**Comment by xFirefalconx** [comment]
Thank you tenfold for the reply. I understand your reasoning and am looking forward to the update then. 
Also noted about the fork. Not everybody likes that so I tend to ask for permission before I do anything. 👍 

Please do be so kind and notify me once the fix is implemented (especially on 1.18.2) so I can update and test if my modpack still functions properly afterwards :D (It's a lot of mods so things tend to break occasionally)
Take care!
